src/Helpers/CleanData.js

```diff
@@ -39,9 +40,15 @@ export const cleanPeople = (people, planets) => {
   const peopleValues = Object.values(people);
 
   return peopleValues.reduce((acc, person) => {
-    acc[person.url] = {
-      
-    }
+//     if () {
+//       if planet url matches persome homeworld url, then s
```

Comment:
I have an idea on how we could grab the people's planets using the planet data (that we already have) and not doubling down on data and going through people.